examples/3d/volumetric_fog.rs

```diff
@@ -60,6 +61,44 @@ fn setup(mut commands: Commands, asset_server: Res<AssetServer>) {
             ..default()
         });
 
+    // Add the point light
+    commands
+        .spawn(PointLightBundle {
+            point_light: PointLight {
+                shadows_enabled: true,
+                range: 50.0,
+                color: RED.into(),
+                intensity: 1000.0,
+                ..default()
+            },
+            transform: Transform::from_xyz(-0.3493744, 1.900556, 1.0452124),
+            ..default()
+        })
+        .insert(VolumetricLight);
+
+    // Add the spot light
+    let mut spotlight_transform = Transform::from_xyz(-1.7817883, 3.901562, -2.7141085);
```

Comment:
When defining scenes via code for examples like this, I often find `Transform::look_at` to be much easier to setup intent.